src/metabase/analytics/prometheus.clj

```diff
@@ -194,11 +194,13 @@
 
 (defn- product-collectors
   []
-  ;; Iapetos will use "default" if we do not provide a namespace, so explicitly set `metabase-email`:
+  ;; Iapetos will use "default" if we do not provide a namespace, so explicitly set, e.g. `metabase-email`:
   [(prometheus/counter :metabase-email/messages
                        {:description (trs "Number of emails sent.")})
    (prometheus/counter :metabase-email/message-errors
-                       {:description (trs "Number of errors when sending emails.")})])
+                       {:description (trs "Number of errors when sending emails.")})
+   (prometheus/counter :metabase-metrics/adjust-errors
```

Comment:
Any naming conventions I should follow for the metric name here?

Reply:
Do we want to also count the total number of adjustments attempted? I included a counter for it in a first pass, but backed it out since I don't have a good sense of how often `adjust` will wind up getting called.

Reply:
I suppose the interesting thing to know would be how often it processed a query with metrics (i.e., had to change the query) and how often the invariant that no `:metric` should be in the result failed.

Reply:
I added the "total adjustments attempted" counter and used the suggested names. Easy to change if @dpsutton has other ideas about naming.